robusta_krr/core/runner.py

```
@@ -329,6 +334,7 @@ async def run(self) -> int:
         except Exception as e:
             logger.error(f"Could not load kubernetes configuration: {e}")
             logger.error("Try to explicitly set --context and/or --kubeconfig flags.")
+            publish_error(f"Could not load kubernetes configuration: {e}")
```

Pylint (3.3.7)

[error] No value for argument 'scan_id' in function call

(E1120)